opencensus/plugins/grpc/internal/server_filter.cc

```diff
@@ -114,14 +115,12 @@ void CensusServerCallData::OnDoneRecvInitialMetadataCb(void *user_data,
                                 ? 0
                                 : GRPC_SLICE_LENGTH(sml.census_proto);
 
-    // TODO: Setup GenerateServerContext().
-    GenerateServerContext(
-        absl::string_view(tracing_str, tracing_str_len),
-        absl::string_view(census_str, census_str_len), absl::string_view("", 0),
-        absl::string_view(method_str, calld->method_size_), &calld->context_);
-
-    absl::string_view method(method_str, calld->method_size_);
-    stats::Record({{RpcServerStartedCount(), 1}}, {{kMethodTagKey, method}});
+    GenerateServerContext(absl::string_view(tracing_str, tracing_str_len),
+                          absl::string_view(census_str, census_str_len),
+                          absl::string_view("", 0) /*primary_role=""*/,
```

Comment:
Drop the string_view and just: `/*primary_role="*/ ""`

Reply:
Done.